app/src/main/java/de/danoeh/antennapod/dialog/ItemFilterDialog.java

```diff
@@ -62,6 +70,11 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
                 }
             }
         }
+        resetTextView.setEnabled(!getNewFilterValues().isEmpty());
```

Comment:
Don't need to look at the new values here

```suggestion
        resetTextView.setEnabled(!filter.getValues().isEmpty());
```